libs/vkd3d/utils.c

```diff
@@ -1067,88 +1067,50 @@ HRESULT vkd3d_get_private_data(struct vkd3d_private_store *store,
         const GUID *tag, unsigned int *out_size, void *out)
 {
     const struct vkd3d_private_data *data;
-    HRESULT hr = S_OK;
     unsigned int size;
-    int rc;
 
     if (!out_size)
         return E_INVALIDARG;
 
-    if ((rc = pthread_mutex_lock(&store->mutex)))
```

Comment:
Do all callers lock outside of vkd3d_get_private_data?

Reply:
Nope, this was a regression from when I changed this to use callbacks from the old change and missed it. Fixed.